frontend/src/components/Contexts/AuthProvider.tsx

```diff
@@ -6,14 +6,22 @@ type Props = {
     children?: React.ReactNode
 }
 
-export const AuthContext = createContext('')
+const defaultAuthState = {
+    accessToken: undefined,
+}
+
+interface IAuthContext {
+    accessToken: string | undefined
+}
+
+export const AuthContext = createContext<IAuthContext>(defaultAuthState)
 
 // Check for new token every second (Will refresh token if needed)
 export const tokenReverificationInterval: number = 1000
 
 export const AuthProvider = (props: Props) => {
     const msalContext = useMsal()
-    const [accessToken, setAccessToken] = useState('')
+    const [accessToken, setAccessToken] = useState<string | undefined>(undefined)
```

Comment:
Avoids relying on empty strings, and instead uses the Typescript type checking to see if the value has been set.